modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogServiceImpl.java

```diff
@@ -877,4 +886,24 @@ private static void validateAlterTableColumn(
             throwUnsupportedDdl("Cannot decrease precision to {} for column '{}'.", target.precision(), origin.name());
         }
     }
+
+    private static CreateHashIndexParams createPkIndexParams(CreateTableParams params) {
+        return CreateHashIndexParams.builder()
+                .schemaName(params.schemaName())
+                .tableName(params.tableName())
+                .indexName(params.tableName() + "_PK")
+                .columns(params.primaryKeyColumns())
+                .unique()
+                .build();
+    }
+
+    private static CatalogHashIndexDescriptor createHashIndexDescriptor(
```

Comment:
Are you sure that extracting this method helps the code readability? I don't think so:
- it's only two lines
- it's only used twice
- it breaks code flow by introducing different name and usage pattern (see one of the comments above)

What if we don't introduce it?

Reply:
I don't see a problem here.